documentation/modules/operators/con-topic-replication.adoc

```diff
@@ -29,4 +29,15 @@ For more information, see xref:proc-cruise-control-topic-replication-{context}[]
 
 NOTE: In-sync replicas are used in conjunction with the `acks` configuration for producer applications.
 The `acks` configuration determines the number of follower partitions a message must be replicated to before the message is acknowledged as successfully received.
-The bidirectional Topic Operator runs with `acks=all` for its internal topics whereby messages must be acknowledged by all in-sync replicas.
+
+When scaling Kafka clusters by adding or removing brokers, replication factor configuration is not changed and replicas are not reassigned automatically.
+However, you can use the `kafka-reassign-partitions.sh` tool to change the replication factor, and manually reassign replicas to brokers.
```

Comment:
I wonder if we should just link to this section on Scaling Clusters, which has the same info: https://strimzi.io/docs/operators/in-development/deploying#con-scaling-kafka-clusters-str